tests/unit/amo/components/TestHeader.js

```diff
@@ -77,17 +76,15 @@ describe(__filename, () => {
 
   it('allows a signed-in user to log out', () => {
     const { store } = dispatchSignInActions({ username: 'babar' });
-    const wrapper = renderHeader({ store });
-    const mockApi = sinon.mock(api);
+    const handleLogOut = sinon.stub();
 
-    mockApi
-      .expects('logOutFromServer')
-      .once()
-      .returns(Promise.resolve());
+    const wrapper = renderHeader({ store, handleLogOut });
 
     const onClick = wrapper.find(DropdownMenuItem).last().prop('onClick');
```

Comment:
Using `.last()` seems a bit fragile. You could give it a class like `Header-logOut` and find that instead.